smqtk_descriptors/impls/descriptor_generator/pytorch.py

```diff
@@ -175,10 +174,7 @@ def __init__(
         super().__init__()
 
         # Make sure given image reader is valid
-        try:
-            image_reader.is_valid_element
-        except AttributeError:
-            raise
+        assert image_reader is not None
```

Comment:
I guess to rephrase my previous question: why are we checking the input `image_reader` value at all? I don't understand or am missing what the failure-case is that we are protecting against here.